datadog/resource_datadog_integration_aws.go

```diff
@@ -83,6 +83,18 @@ func resourceDatadogIntegrationAws() *schema.Resource {
 				ConflictsWith: []string{"account_id", "role_name"},
 				Optional:      true,
 			},
+			"metrics_collection_enabled": {
+				Description: "Whether Datadog collects metrics for this AWS account.",
+				Type:        schema.TypeBool,
+				Default:     false,
```

Comment:
```suggestion
				Default:     true,
```
Let's make this option default to `true` in order to keep it in line with the api and the datadog go client behavior.